### PR TITLE
Event Manager split interface

### PR DESCRIPTION
**Pull request by gianarb**
This commit contains set of interfaces to support EventManager
replacement.

Penny\Event\ZendEvmProxy will be moved in a separate repository to
remove forced  Zend\EventManager dependency from penny

@pennyphp/community-review @wdalmut

### CODE REVIEW

**Comment by gianarb** [comment]
Reference https://github.com/pennyphp/penny-skeleton-app/issues/40#issuecomment-155690133

File: src/Event/ZendEvmProxy.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class ZendEvmProxy implements PennyEvmInterface
+{
+    private $eventManager;
+
+    public function __construct()
+    {
+        $this->eventManager = new EventManager();
+    }
+
+    public function trigger(PennyEventInterface $event)
```
Comment (samsonasik) [inline src/Event/ZendEvmProxy.php:24]: missing docblocks

**Comment by gianarb** [comment]
@samsonasik what do you think about this feature?

**Comment by samsonasik** [comment]
it's good +1

**Comment by gianarb** [comment]
Thanks @samsonasik! :)

**Pull request merged**